plugins/CorePluginsAdmin/angularjs/plugins/plugin-management.directive.js

```diff
@@ -25,6 +25,23 @@
 
                     var uninstallConfirmMessage = '';
 
+                    $('.uploadPlugin').click(function (event) {
```

Comment:
As this code is duplicated in plugins/Marketplace/angularjs/marketplace/marketplace.directive.js - could we refactor it in the new  plugins/Marketplace/templates/uploadPluginDialog.twig  ?